facilitator/src/logging.rs

```diff
@@ -0,0 +1,57 @@
+use env_logger::fmt::Formatter;
+use log::{Level, Record};
+use serde::Serialize;
+use std::io::Write;
+
+#[derive(Debug, Serialize)]
+#[serde()]
```

Comment:
I think this should be `#[serde(rename_all = "UPPERCASE")]` and then the variatns should be named according to Rust convention. The enum should also have a doccomment explaining why it exists -- it's to map `log::Level` to GCP's notion of log severity, right?